ts/parts-more/BubbleSeries.ts

```diff
@@ -468,7 +474,8 @@ seriesType<Highcharts.BubbleSeriesOptions>('bubble', 'scatter', {
             zMin = 0;
         }
 
-        if (!isNumber(value)) {
+        // #8608 - bubble should be visible when z is undefined
+        if (yValue === null || value === null) {
```

Comment:
Now that BubbleSeries is migrated to TypeScript this condition here becomes redundant in my understanding.

Since `value` is required and can only be a `number` and thus never `null`.
Since `yValue` is optional it can only be `number` or `undefined` and thus never `null`

Because of this TypeScript should alert us when we try to pass in `yValue` or `value` with the type of `null`, and we could in my opinion remove this condition.

Reply:
IMHO its safer to leave this condition.

What do you think @oysteinmoseng @pawelfus ?

Reply:
TS defs are wrong:
> public yData: Array<number>;

Should be:
> public yData: Array<(number|undefined|null)>;

The same for `zData`.

Demo: https://jsfiddle.net/BlackLabel/ek83tch9/

Reply:
Also, the TypeScript defs for `getRadius` has to be changed to `value: number|null|undefined` and `yValue?: number|null` for this to be resolved. When this is changed the casting of `value` and `yValue` to `any` in `getRadii` can be removed again.